package.json

```diff
@@ -1,43 +1,58 @@
-	"version": "5.0.0",
-	"description": "WordPress is web software you can use to create a beautiful website or blog.",
-	"repository": {
-		"type": "svn",
-		"url": "https://develop.svn.wordpress.org/trunk"
-	},
-	"engines": {
-		"node": "8.9.3"
-	},
-	"author": "The WordPress Contributors",
-	"license": "GPL-2.0-or-later",
-	"devDependencies": {
-		"autoprefixer": "^6.5.1",
-		"grunt": "~0.4.5",
-		"grunt-banner": "^0.6.0",
-		"grunt-contrib-clean": "~1.0.0",
-		"grunt-contrib-compress": "~1.3.0",
-		"grunt-contrib-concat": "~1.0.0",
-		"grunt-contrib-copy": "~1.0.0",
-		"grunt-contrib-cssmin": "~1.0.2",
-		"grunt-contrib-imagemin": "~1.0.0",
-		"grunt-contrib-jshint": "~1.0.0",
-		"grunt-contrib-qunit": "^1.2.0",
-		"grunt-contrib-uglify": "~2.0.0",
-		"grunt-contrib-watch": "~1.0.0",
-		"grunt-includes": "~0.5.1",
-		"grunt-jsdoc": "^2.1.0",
-		"grunt-jsvalidate": "~0.2.2",
-		"grunt-legacy-util": "^0.2.0",
-		"grunt-patch-wordpress": "~0.4.2",
-		"grunt-postcss": "~0.7.1",
-		"grunt-replace": "~1.0.1",
-		"grunt-rtlcss": "~2.0.1",
-		"grunt-sass": "2.0.0",
-		"ink-docstrap": "^1.3.0",
-		"grunt-webpack": "^3.0.2",
-		"matchdep": "~1.0.0",
-		"webpack": "^3.6.0",
-		"webpack-dev-server": "^2.9.1"
-	}
+  "name": "WordPress",
+  "version": "5.0.0",
+  "description": "WordPress is web software you can use to create a beautiful website or blog.",
+  "repository": {
+    "type": "svn",
+    "url": "https://develop.svn.wordpress.org/trunk"
+  },
+  "engines": {
+    "node": "8.9.3"
+  },
+  "author": "The WordPress Contributors",
+  "license": "GPL-2.0-or-later",
+  "devDependencies": {
+    "autoprefixer": "^6.5.1",
+    "grunt": "~0.4.5",
+    "grunt-banner": "^0.6.0",
+    "grunt-contrib-clean": "~1.0.0",
+    "grunt-contrib-compress": "~1.3.0",
+    "grunt-contrib-concat": "~1.0.0",
+    "grunt-contrib-copy": "~1.0.0",
+    "grunt-contrib-cssmin": "~1.0.2",
+    "grunt-contrib-imagemin": "~1.0.0",
+    "grunt-contrib-jshint": "~1.0.0",
+    "grunt-contrib-qunit": "^1.2.0",
+    "grunt-contrib-uglify": "~2.0.0",
+    "grunt-contrib-watch": "~1.0.0",
+    "grunt-file-append": "0.0.7",
+    "grunt-includes": "~0.5.1",
+    "grunt-jsdoc": "^2.1.0",
+    "grunt-jsvalidate": "~0.2.2",
+    "grunt-legacy-util": "^0.2.0",
+    "grunt-patch-wordpress": "~0.4.2",
+    "grunt-postcss": "~0.7.1",
+    "grunt-replace": "~1.0.1",
+    "grunt-rtlcss": "~2.0.1",
+    "grunt-sass": "2.0.0",
+    "grunt-webpack": "^3.0.2",
+    "ink-docstrap": "^1.3.0",
+    "jquery-migrate": "1.4.1",
+    "matchdep": "~1.0.0",
+    "webpack": "^3.6.0",
+    "webpack-dev-server": "^2.9.1"
+  },
+  "dependencies": {
+    "backbone": "1.3.3",
+    "imagesloaded": "3.2.0",
+    "jquery": "1.12.4",
+    "jquery-color": "github:jquery/jquery-color#2.1.1",
+    "jquery-form": "4.2.1",
+    "jquery-hoverintent": "1.8.3",
+    "jquery-ui": "github:jquery/jquery-ui#1.11.4",
+    "masonry-layout": "3.3.2",
+    "twemoji": "2.4.0",
+    "underscore": "1.8.3",
+    "zxcvbn": "4.4.1"
+  }
 }
```

Comment:
The `package.json` file switched to using tabs for indentation in [[42460](https://core.trac.wordpress.org/changeset/42460/)]

After updating `package.json` to tabs, `package-lock.json` should be deleted and regenerated via `npm install` so that it inherits the _tabbed_ indentation from `package.json`